pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -369,6 +373,11 @@ type AlertmanagerWebSpec struct {
 	Timeout *uint32 `json:"timeout,omitempty"`
 }
 
+// TODO: What other fields do we need?
+type AlertmanagerMTLSConfig struct {
+	MTLSConfigFields `json:",inline"`
```

Comment:
I'm not sure why we would need an inline field.

```suggestion
	// Server-side configuration for mutual TLS.
	Server *ServerTLSConfig `json:"server"`
	// Client-side configuration for mutual TLS.
	Client *ClientTLSConfig `json:"client"`
```